ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -94,24 +140,74 @@ tests:
   container:
     from: src
   run_if_changed: ^(rpm/*|packaging/.*|Makefile)
-- as: e2e-openshift-conformance-reduced
+- as: rebase-on-nightlies
+  cron: 0 5 * * 1-5
+  steps:
+    workflow: openshift-microshift-rebase
+- as: e2e-openshift-conformance-reduced-rhel8-x86-64
+  cluster: build02
+  run_if_changed: ^(assets/.*|cmd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
+  steps:
+    cluster_profile: gcp
+    env:
+      GOOGLE_COMPUTE_IMAGE_NAME: rhel-8-v20230202
+      GOOGLE_COMPUTE_MACHINE_TYPE: e2-standard-8
```

Comment:
What do you think about not overwriting machine types so that future changes to it dont impact these configs?
It's not like we know/need specific requirements for any of these jobs as of today, so best keep it simple.